packages/rx_bloc/lib/src/annotation/rx_bloc_annotations.dart

```diff
@@ -66,7 +66,7 @@ class RxBlocEvent {
   final RxBlocEventType type;
 
   /// The initial value of the event
-  final dynamic? seed;
+  final dynamic seed;
```

Comment:
@DDavidPrime, hmmm I'm not sure about this, though. Check [this](https://github.com/Prime-Holding/rx_bloc/issues/23) issue.

Reply:
Consider this screenshot:

![pre](https://i.ibb.co/kDCzKd3/Screenshot-2021-05-21-at-16-39-26.png)

From the screenshot above we can see that there's a message saying `'?' is unnecessary because 'dynamic' is nullable
`. This would mean that it accepts non-nullable and nullable values.

(The preview can be found [here](https://github.com/Prime-Holding/rx_bloc/runs/2630322147), in the failing tests).